src/Core/src/Handlers/GraphicsView/GraphicsViewHandler.Windows.cs

```diff
@@ -24,5 +39,10 @@ public static void MapInvalidate(IGraphicsViewHandler handler, IGraphicsView gra
 		{
 			handler.PlatformView?.Invalidate();
 		}
+
+		void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			VirtualView?.InvalidateMeasure();
```

Comment:
Can you explain the fix? should't this be called internally why is it different for the GraphicsView ?

Reply:
Between GA and SR1, GraphicsView does not call the Drawable's Draw method. The changes in this PR are to force a Measure when loading the control to fix the problem. Probably, the origin of the issue comes from ContentView and some change in Layout but I have focused the changes on GraphicsView to ensure the fix and avoid side effects.